packages/manager/modules/pci/src/projects/project/storages/object-storage/users/import/import.routing.js

```diff
@@ -0,0 +1,23 @@
+import find from 'lodash/find';
+
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state(
+    'pci.projects.project.storages.objects.users.import-policy',
+    {
+      url: '/:userId/import-policy',
```

Comment:
```suggestion
      url: '/import-policy?userId',
```
As there is no user dashboard or if not state should be a child of that dashboard